docs/commands/SetParameters.md

```diff
@@ -0,0 +1,81 @@
+# SetParameters
+
+Parameters setup command. The sensor confirms the execution of the command with [response](#response)
+
+
+## Request
+
+### Format
+
+> Size | Type | Field
+> -----|------|-------
+> 1    | byte | command id = `0x03`
+> 1    | byte | command dynamic size
+> 1    | byte | [parameter type](#parameter-type)
+> 1+   | byte | [parameter data](#parameter-data)
+
+### Parameters
+
+#### **parameter type**
+
+See [available parameter types](../parameter-types.md)
+
+#### **parameter data**
+
+Type-specific structures.
+
+### Examples
```

Comment:
maybe examples in parameter-types.md would be enougth?

Reply:
maybe, this example removed